ui/raidboss/timeline.js

```diff
@@ -787,29 +787,76 @@ class TimelineUI {
   }
 }
 
+const CountdownState = {
+  none: 0,
+  active: 1,
+  wipe: 2,
+};
+
 class TimelineController {
   constructor(options, ui) {
     this.options = options;
     this.ui = ui;
     this.dataFiles = {};
     // data files not sent yet.
     this.timelines = null;
+
+    // Fix for #1453
+    // Countdown state
+    // 0 = No countdown
+    // 1 = Countdown active
+    // 2 = Wiped during countdown
+    this.countdownState = CountdownState.none;
+    this.wipeRegex = Regexes.network6d({ command: '40000010' });
   }
 
   SetPopupTextInterface(popupText) {
     this.ui.SetPopupTextInterface(popupText);
   }
 
   SetInCombat(inCombat) {
+    // On wipe, set countdown state to Wipe if it was Active
+    if (!inCombat && this.countdownState === CountdownState.active)
+      this.countdownState = CountdownState.wipe;
     if (!inCombat && this.activeTimeline)
       this.activeTimeline.Stop();
   }
 
   OnLogEvent(e) {
     if (!this.activeTimeline)
       return;
-    for (let i = 0; i < e.detail.logs.length; ++i)
+    for (let i = 0; i < e.detail.logs.length; ++i) {
+      let startMatch =
+        e.detail.logs[i].match(LocaleRegex.countdownStart[this.options.ParserLanguage]);
+      if (startMatch) {
+        // If we're not in a wipe state, reset the state to countdown active
+        if (this.countdownState !== CountdownState.wipe)
+          this.countdownState = CountdownState.active;
```

Comment:
Should this section unconditionally set it to `active`? If you wipe and then start another countdown, that should work ok?

Reply:
The countdown regex will match on every 5 second interval of the countdown timer. Consider this set of events and the corresponding state:

1. 20s countdown starts, `CountdownState.active`
2. 15s line, `CountdownState.active`
3. wipe, `CountdownState.wipe`
4. 10s line, `CountdownState.active`
5. 5s line, `CountdownState.active`
6. 3/2/1, not sure if these match the regex
7. Engage!, timeline starts despite wipe because state is `CountdownState.active`

Thus, the check for Engage or Cancel is what clears the state back to `CountdownState.none`. It could be rewritten to check `this.countdownState === CountdownState.none`, but some sort of check is required here.

Reply:
Oh, that's my misunderstanding.  I thought the regex included the person's name, which only appears the first time.  (It probably should, but it doesn't.)  Got it, I see this case then.  Ok, then I guess you need to do it conditionally + watch for cancelling + zone changing clearing too?

Reply:
Yeah, need all three. I tested this in-game, it's one of the things I didn't catch on first pass and had to add in.